### PR TITLE
French translation - FAQ section

### PR DESCRIPTION
**Pull request by Gimly**
## Description

This PR adds the translated files for the FAQ section of the documentation in French.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* #6924

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by noroadsleft** [comment]
@Gimly As detailed on https://github.com/qmk/qmk_firmware/pull/6966#issuecomment-540237003, can you use `docs/fr-fr/` as the directory, please? This will also need a rebase at some point. But you can do that when you're done with the actual translating.

Requesting review from @zekth.

**Comment by Gimly** [comment]
OK, I'm done with this part of the translation, this is ready for review.

@noroadsleft I've moved all the files from the fr-FR folder to fr-fr folder, sorry about the issue.

**Comment by 4sStylZ** [comment]
Seem’s that this issue will fixe #7002 because of the file renamed.

**Comment by Gimly** [comment]
@4sStylZ maybe you can help reviewing this pr? Thanks.

**Comment by 4sStylZ** [comment]
> @4sStylZ maybe you can help reviewing this pr? Thanks.

Yep, it's on my todolist 🐒

**Comment by 4sStylZ** [comment]
I have done the whole review of this but I can't submitting because Github timeout…

If someone have a suggestion please go.

**Comment by 4sStylZ** [comment]
Ok I hate github now and I have submitted all the code of my review on the Gimly github.

I suggest that nobody review this PR for now since he need to correct my suggestionsr or refuse them ;)

**Comment by Gimly** [comment]
The changes linked to the review from @4sStylZ have been committed to the PR's branch. Maybe another review?

**Comment by 4sStylZ** [comment]
If you can take a look please @zekth 

Thank you

**Comment by drashna** [comment]
Looks like there is a merge conflict here.  If you could resolve that?

**Comment by Gimly** [comment]
@drashna rebased on the current master, should be OK now

File: docs/fr-fr/faq_build.md [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+brew install avr-gcc
+brew install dfu-programmer
+brew install dfu-util
+brew install gcc-arm-none-eabi
+brew install avrdude
+```
+
+### avr-gcc 8.1 et LUFA
+
+Si vous avez mis à jour votre avr-gcc au dessus de la version 7, vous risquez de voir des erreurs impliquant LUA. Par exemple :
```
Comment (Ph0tonic) [inline docs/fr-fr/faq_build.md]: `Au-dessus` prend un tiret.

File: docs/fr-fr/faq_build.md [hunk 2]
```diff
@@ -0,0 +1,182 @@
+# Foire aux questions sur la compilation
+
+Cette page couvre les questions concernant la compilation de QMK. Si vous ne l'avez pas encore fait, vous devriez lire les guides [Configuration de l'environnement de build](getting_started_build_tools.md) et [Instructions pour Make](getting_started_make_guide.md).
+
+## Je ne peux pas programmer sous Linux
+
+Vous aurez besoin des permissions appropriées pour utiliser un périphérique. Pour les utilisateurs de Linux, référez-vous aux instructions concernant les règles `udev` ci-dessous. Si `udev` vous pose des problèmes, une alternative est d'utiliser la commande `sudo`. Si vous ne connaissez pas cette commande, référez vous à son manuel d'utilisation en utilisant `man sudo` ou [regardez cette page](https://linux.die.net/man/8/sudo).
```
Comment (Ph0tonic) [inline docs/fr-fr/faq_build.md]: Il manque un tiret pour `référez-vous`.

File: docs/fr-fr/faq_build.md [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+
+    $ sudo dfu-programmer atmega32u4 erase --force
+    $ sudo dfu-programmer atmega32u4 flash your.hex
+    $ sudo dfu-programmer atmega32u4 reset
+
+ou simplement :
+
+    $ sudo make <keyboard>:<keymap>:dfu
+
+Veuillez noter que lancer `make` avec `sudo` est généralement une **mauvaise** idée, et vous devriez préférez une des méthodes précédente, si possible.
```
Comment (Ph0tonic) [inline docs/fr-fr/faq_build.md]: `préférer` au lieu de `préférez`.

File: docs/fr-fr/faq_build.md [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+SUBSYSTEMS=="usb", ATTRS{idVendor}=="0483", ATTRS{idProduct}=="df11", MODE:="0666"
+```
+
+### Le périphérique sériel n'est pas détecté en mode bootloader sous Linux
+
+Assurez-vous que votre kernel ait un support approprié pour votre périphérique. Si votre périphérique utilise USB ACM, par exemple pour les Pro Micro (AtMega32u4), assurez-vous d'inclure `CONFIG_USB_ACM=y`. D'autres périphériques peuvent avoir besoin de `USB_SERIAL` et de ses sous-options.
+
+## Périphérique inconnu pour le bootloader DFU
+
+Les problèmes rencontrés lorsque l'on flash des claviers sous Windows sont, la plupart du temps, dûs à une installation du mauvais pilote, ou un pilote manquant.
```
Comment (Ph0tonic) [inline docs/fr-fr/faq_build.md]: `dus` ne prend pas d'accent circonflexe au présent pluriel.

File: docs/fr-fr/faq_debug.md [hunk 5]
```diff
@@ -0,0 +1,236 @@
...
+- Les outils DFU **ne** vous permettent **pas** d'écrire dans le bootloader (à moins que vous n'ajoutiez des options spéciales), il n'y a donc peu de risque.
+- Les EEPROM ont environ 100000 cycles d'écriture. Ne réécrivez pas le firmware de manière continue et répétée. Vous allez détruire définitivement l'EEPROM.
+
+## NKRO ne fonctionne pas
+
+Premièrement, vous devez compiler le firmware avec l'option de compilation `NKRO_ENABLE` dans le **Makefile**.
+
+Essayez la commande `Magic` **N** (`LShift+RShift+N` par défaut) si **NKRO** ne fonctionne toujours pas. Vous pouvez utiliser cette commande pour basculer temporairement entre le mode **NKRO** et **6KRO**. Sous certaines conditions, **NKRO** ne fonctionnera pas et vous devrez basculer en **6KRO**, en particulier lorsque vous êtes dans le BIOS.
+
+Si votre firmware est compilé avec `BOOTMAGIC_ENABLE` vous devrez l'activer avec la commande `BootMagic` **N** (`Espace+N` par défaut). Cette option est enregistrées dans l'EEPROM et sera gardé entre deux cycle de démarrage.
```
Comment (Ph0tonic) [inline docs/fr-fr/faq_debug.md]: Problème avec l'accord du verbe enregistrer qui devrait être `enregistrée`.

Comment (Ph0tonic) [reply docs/fr-fr/faq_debug.md]: Manque un `s` à `cycles`.

File: docs/fr-fr/faq_keymap.md [hunk 6]
```diff
@@ -0,0 +1,226 @@
...
+Cette fonctionnalité permet l'usage de *touches à verrouillage mécanique* comme [ces interrupteurs Alps](http://deskthority.net/wiki/Alps_SKCL_Lock). Vous pouvez l'activer en ajoutant ceci à votre `config.h` :
+
+```
+#define LOCKING_SUPPORT_ENABLE
+#define LOCKING_RESYNC_ENABLE
+```
+
+Une fois la fonction activée, utilisez les keycodes `KC_LCAP`, `KC_LNUM` et `KC_LSCR` dans votre keymap.
+
+Des vieux claviers mécaniques ont parfois des touches à vérouillage, mais les claviers modernes n'en sont pas équipés. ***Vous n'avez pas besoin de cette fonction dans la majorité des cas et devez utiliser les keycodes `KC_CAPS`, `KC_NLCK` et `KC_SLCK`.***
```
Comment (Ph0tonic) [inline docs/fr-fr/faq_keymap.md]: `verrouillage` au lieu de `vérouillage`.

File: docs/fr-fr/faq_keymap.md [hunk 7]
```diff
@@ -0,0 +1,226 @@
...
+
+Voir ce post pour un exemple de code **MACRO**.
+
+http://deskthority.net/workshop-f7/tmk-keyboard-firmware-collection-t4478-120.html#p195620
+
+Sous **Windows** vous pouvez utiliser la touche `AltGr` ou **Alt code**.
+* http://en.wikipedia.org/wiki/AltGr_key
+* http://en.wikipedia.org/wiki/Alt_code
+
+Sous **Mac OS** définissez une compinaison de touche `Option`.
```
Comment (Ph0tonic) [inline docs/fr-fr/faq_keymap.md]: `combinaison` au lieu de `compinaison`

File: docs/fr-fr/faq_keymap.md [hunk 8]
```diff
@@ -0,0 +1,226 @@
...
+#### KC_MNXT et KC_MPRV ne fonctionnent pas sous Mac
+
+Utilisez `KC_MFFD`(`KC_MEDIA_FAST_FORWARD`) et `KC_MRWD`(`KC_MEDIA_REWIND`) à la place de `KC_MNXT` et `KC_MPRV`.
+Voir https://github.com/tmk/tmk_keyboard/issues/195
+
+## Touches supportées sous Mac OSX?
+
+Vous pouvez connaître les keycodes supportés par OSX en lisant ce code source.
+
+`usb_2_adb_keymap` contient les tableaux des page Keyboard/Keypad vers les scancodes ADB (keycodes interne à OSX).
```
Comment (Ph0tonic) [inline docs/fr-fr/faq_keymap.md]: Problème d'accord pour `des pages`.

File: docs/fr-fr/faq_keymap.md [hunk 9]
```diff
@@ -0,0 +1,226 @@
...
+* Activer la touche KATAKANA sur clavier PC
+
+https://pqrs.org/osx/karabiner/seil.html
+
+## RN-42 Bluetooth ne fonctionne pas avec Karabiner
+
+Karabiner - Outil de Keymapping sous Mac OSX - Ignore les entrées du module RN-42. Vous devez activer cette option pour rendre Karabiner compatible avec votre clavier.
+https://github.com/tekezo/Karabiner/issues/403#issuecomment-102559237
+
+Plus de détails sur ce problèmes sur les liens suivants.
```
Comment (Ph0tonic) [inline docs/fr-fr/faq_keymap.md]: Un `s` en trop pour `problème`.

**Comment by Gimly** [comment]
@Ph0tonic Thanks a lot for the review, fixed all the comments

@drashna I think it should be good to be merged now

**Comment by noroadsleft** [comment]
@Gimly Can you rebase this branch onto a current master again please? Sorry.

*noroadsleft mumbles something about too many PRs to keep in his head.*

**Comment by Gimly** [comment]
@noroadsleft I've rebased the branch, should be OK.

That's the Hacktoberfest effect for the number of PRs 😄

**Comment by noroadsleft** [comment]
Thanks!

**Comment by Gimly** [comment]
Glad I could help 😄

**Pull request merged**